android/modules/ui/src/java/ti/modules/titanium/ui/widget/webview/TiUIWebView.java

```diff
@@ -264,7 +264,16 @@ public boolean onCheckIsTextEditor()
 			return super.onCheckIsTextEditor();
 		}
 	}
-	
+
+	@Override
+	protected boolean hasBorder(KrollDict d) {
```

Comment:
minor, but a warning could be thrown suggesting the method does not return a value (even though it does). You should change it to this:
```Java
protected boolean hasBorder(KrollDict d) {
	if (LOWER_THAN_JELLYBEAN) {
		return false;
	}
	return super.hasBorder(d);
}
```

Reply:
Updated.